### DISCUSSION TITLE
[resolved] nr zamówienia w billlingu

### DISCUSSION THREAD

**Discussion by drzemik** [allegro/allegro-api#4334] [Suggestions - resolved]
Jak uzystkać listę prowizji powiązaną z zamówieniami?
Historia operacji billingowych w zasobie GET /billing/billing-entries
niestety nie zawiera tej informacji, a jedynie numer oferty.

**Comment by Lukasz-Zurek** [comment]
Nie uzyskasz listy prowizji posiadając numer zamówienia. Nie udostępniamy tego pola we wskazanym zasobie.

Mamy już jednak taką sugestię na liście dot. rozwoju REST API - dodaję Twój głos.

**Comment by AureliuszBrussy** [comment]
Podjeliśmy decyzję, że zwrócimy pole, w którym wskażemy numer zamówienia, z którym powiązana jest opłata. O szczegółach będziemy jeszcze informować.

**Reply by jakubmaguza** [reply]
Czy jest jakiś przewidywany termin tej zmiany?

**Reply by AureliuszBrussy** [reply]
Powinniśmy wdrożyć zmianę w ciągu najbliższych tygodni.

**Comment by AureliuszBrussy** [comment]
Wdrożyliśmy już tą sugestię. Więcej informacji w [naszym newsie](https://github.com/allegro/allegro-api/issues/4528).
